blueprints/data-solutions/vertex-mlops/variables.tf

```diff
@@ -0,0 +1,228 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+variable "billing_account_id" {
+  description = "Billing account id."
+  type        = string
+}
+
+variable "bucket_name" {
+  description = "Create GCS Bucket."
+  type        = string
+  default     = null
+}
+
+variable "dataset_name" {
+  description = "Create BigQuery Datasets."
+  type        = string
+  default     = null
+}
+
+variable "env" {
+  description = "Environment (dev,stg,prd)."
+  type        = string
+  default     = "dev"
+}
+
+variable "folder_id" {
+  description = "Folder ID for the folder where the project will be created."
+  type        = string
+}
+
+variable "groups" {
+  description = "User groups."
+  type        = map(string)
+  default = {
+    data-scientists = "gcp-ml-ds"
+    ml-engineers    = "gcp-ml-eng"
+  }
+}
+
+variable "identity_pool_claims" {
+  description = "Claims to be used by Workload Identity Federation (i.e.: attribute.repository/ORGANIZATION/REPO)."
+  type        = string
+  default     = null
+}
+
+variable "kms_service_agents" {
+  description = "KMS IAM configuration in as service => [key]."
+  type        = map(list(string))
+  default     = {}
+}
+
+variable "labels" {
+  description = "Labels to be assigned at project level."
+  type        = map(string)
+  default     = {}
+}
+
+
+variable "notebooks" {
+  description = "Vertex AI workbenchs to be deployed."
+  type = map(object({
+    owner                 = string
+    region                = string
+    subnet                = string
+    internal_ip_only      = bool
```

Comment:
Do we want to expose this variable? do you see a real ent customer scenario where they want to use public IPs?

Reply:
I've put this as optional, with true default